spec.md

```diff
@@ -676,9 +676,8 @@ message ValidateVolumeCapabilitiesRequest {
   // The API version assumed by the CO. This is a REQUIRED field.
   Version version = 1;
 
-  // The information about the volume to check. This is a REQUIRED
-  // field.
-  VolumeInfo volume_info = 2;
+  // The ID of the volume to check. This is a REQUIRED field.
+  VolumeID volume_id = 2;
```

Comment:
Discussed with @jieyu: for now `VolumeInfo` lets a plugin check access mode, which is a requirement. Let's not make this change here for now.